source/community/release_notes/redwood.rst

```diff
@@ -13,6 +13,47 @@ These are the release notes for the Redwood release, the 18th community release
 Breaking Changes
 ****************
 
+-  Deployers must ensure that their
+   ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']`` Django setting in LMS
+   contains the full complement of private key numbers.*
+
+   -  **Background:**\ *In LMS, we are switching from the pyjwkest
+      library to PyJWT for signing JWTs. (pyjwkest is now unmaintained.)
+      However, PyJWT has stricter requirements for the private key in
+      ``JWT_PRIVATE_SIGNING_JWK``. Before you upgrade to Redwood, you
+      will need to update this key using a script. Otherwise, JWT
+      signing will fail, and users will be unable to log in.*
+
+   -  **Steps:**
+
+      1. Locate ``JWT_PRIVATE_SIGNING_JWK`` in your deployment
+         configuration.*
```

Comment:
Each of the bullets in this section ends with an asterisk. Does that have some meaning?

Reply:
Ditto conversion artifact.